### ISSUE TITLE
CUDA error: illegal memory access (on Codalab)

### ISSUE THREAD

**Issue by Gbellport** [AlexeyAB/darknet#3410]
I have been having the following issue while working on Codalab (CUDA version and Makefile config shown on pic):
![issue3](https://user-images.githubusercontent.com/19672976/59479895-24455080-8e24-11e9-92ef-1c7b130bfb51.PNG)
When using cuda-memcheck I get multiple blocks of the following issue:
![issue2](https://user-images.githubusercontent.com/19672976/59479973-76867180-8e24-11e9-92a2-b3c9fdbe4d39.PNG)
I quite frankly cannot understand what the issue is, but I have tried the following things without being able to fix the issue: 
- Setting CUDNN=1 in the Makefile 
- Training both yolov3-tiny and yolov3
- Trying various height and width values (only multiples of 32)
- EDIT: tried with CUDA 10.1
Could it be an issue with the GPU that Codalab uses?

**Comment by AlexeyAB** [comment]
What GPU is used on Codalab?
What cuDNN version do you use?

Can you show output of commands?
```
nvcc --version
nvidia-smi
```

**Comment by Gbellport** [comment]
Hey Alexey, so I actually managed to train the model by using an older copy of your repository (about 6 months old). Nevertheless I will answer your questions: 

- GPU? I believe it is a Nvidia Tesla K80 
- cuDNN? version 7.1.4
- The error occurred only during training, I did manage to use detection on the test examples
- I did try setting CUDNN=1 and the error was the exact same as in the screenshot above. The only thing that changed every time was the location of the error in the GPU
- The first screenshot shows the output of !nvcc --version

Hope you manage to figure out what might be causing these issues, it seems to be happening with specific GPUs only but again, it's not happening in older versions of the repo.
